dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/maatapi/MaatApiWebClientFactory.java

```diff
@@ -49,6 +67,7 @@ public WebClient maatApiWebClient(
             .filter(addLaaTransactionIdToRequest())
             .filter(logClientResponse())
             .filter(handleErrorResponse())
+                .filter(new WebClientMetricsFilter(meterRegistry, MAAT_API_WEBCLIENT_REQUESTS))
```

Comment:
Can we have a unit test for this (i.e. make a few calls and check that the counter is incremented as expected)?